dcos/jsonitem.py

```diff
@@ -283,8 +283,9 @@ def _parse_url(value):
     if value_regex is None:
         scheme_match = re.match(scheme_pattern, value, re.IGNORECASE)
         if scheme_match is None:
-            msg = 'Please check url {!r}. Missing http(s)://'.format(value)
-            raise DCOSException(msg)
+            logger.debug("Defaulting URL to https scheme")
+            value = "https://" + value.lstrip(scheme_pattern)
```

Comment:
In addition, I do not understand why we want to "lstrip" the scheme here at all. Using the terminology used in this code section, we just confirmed that the string "value" does not have a "scheme pattern" in its beginning. So why would we try to strip it?